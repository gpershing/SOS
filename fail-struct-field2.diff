diff: tests/fail-struct-field2.err: No such file or directory
