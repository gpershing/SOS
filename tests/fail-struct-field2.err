Fatal error: exception Failure("Cannot access fields for a non-struct variable")
