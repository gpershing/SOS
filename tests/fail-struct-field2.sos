struct s = {int x}

int b = 1

b.x
