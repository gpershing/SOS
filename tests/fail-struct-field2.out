Cannot access fields for a non-struct variable
